internal/generate/gen/rules.go

```diff
@@ -45,12 +45,12 @@ func (o OutputToCachedDirectory) Open(_ *loader.Package, path string) (io.WriteC
 		return nil, fmt.Errorf("error opening %s in output rule: cache must be set", path)
```

Comment:
Anything in `internal` is an internal implementation detail, and errors in here are not necessarily relevant to users. According to the working with errors [blog post](https://blog.golang.org/go1.13-errors), implementation detail errors should not necessarily be exposed for unwrapping. Unless explicitly required by an error consumer within SDK code, `internal` errors should not be wrapped for now. You can still change `errors.Wrap` to `fmt.Errorf("...: %s", ..., err)`, which will hide the error if it implements `Unwrap`.

Reply:
Done